journal/original/20-chapter-17.adoc

```diff
@@ -1054,6 +1080,9 @@ Though by reason of my weakness, I could not travel amongst Friends as I used to
 yet in the motion of life,
 I sent the following lines as an encouraging testimony to them:--
 
+[.embedded-content-document.testimony]
```

Comment:
Lol, don't hate me!  So even though he uses the word testimony directly above, I think we should change this to `epistle`, because he's using the word "testimony" in a more general sense. We want to reserve the `testimony` type for more "official" testimonies about dead people, or for when meetings give a "testimony" about the religious labors of a Friend.

Reply:
lol no worries. :)